doc-Provisioning_Virtual_Machines_and_Hosts/topics/appe-creating-a-service-for-provisioning-vm.adoc

```diff
@@ -0,0 +1,169 @@
+[appendix]
+
+[[example-creating-a-service]]
+== Example: Creating a Service for Provisioning a Virtual Machine through Red Hat Virtualization (RHV)
+
+{product-title} provides the ability to define services, which can include one or more virtual machines or instances, and can be deployed across hybrid environments. Services can be made available to users through the service user interface (SUI) that allows users to order predefined IT services without IT operations getting involved.
+
+The following three items are required to make a service available to users from the {product-title_short} service user interface:
+
+. *Service Dialog*: A service dialog is where you allow users to configure virtual machine or instance options. A service dialog determines which options users get to change. You can choose the options presented to the user. You could just give them the option to set the service name, or you could have them change all of the provisioning dialog options. 
+
+. *Service Catalog*: A service catalog is used to group service dialogs together.
+
+. *Service Catalog Item*: A service catalog item is the actual service which combines a service dialog with a provisioning dialog.
+
+[[provisioning-dialog]]
+=== Examining the Provisioning Dialog
+
+Understanding provisioning dialogs is important to work with services in {product-title_short}. Every provider in {product-title_short} comes with a sample provisioning dialog covering the options specific to that provider. A provisioning dialog presents the basic configuration options for a virtual machine or instance. When you begin the process of provisioning a virtual machine or instance via {product-title_short}, you are presented with a provisioning dialog where you set certain options for the virtual machine or instance. The options presented are dependent on the provider you are using. For instance, a cloud provider might have flavors of instances, whereas an infrastructure provider might allow you to set the memory size or number of CPUs on a virtual machine.
```

Comment:
Here again I might edit down starting with "A provisioning dialog presents the basic configuration" as the sentence that follows presents some redundant information. Maybe something along the lines of "The process begins with the provisioning dialog, which presents the basic configuration options for a virtual machine or instance."

Reply:
Modified, thanks.